src/com/serotonin/mango/Common.java

```diff
@@ -41,8 +36,7 @@
 import org.apache.commons.httpclient.auth.AuthScope;
 import org.apache.commons.httpclient.params.HttpClientParams;
 import org.apache.commons.httpclient.params.HttpConnectionManagerParams;
-import org.directwebremoting.WebContext;
-import org.directwebremoting.WebContextFactory;
+import org.directwebremoting.*;
```

Comment:
Please keep previous imports with the explicit declaration.

Reply:
ok, thank you